BAZProject/MovieDetail/MovieDetailProtocols.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  MovieDetailProtocols.swift
+//  BAZProject
+//
+//  Created 1030364 on 17/10/22.
+//  Copyright © 2022 ___ORGANIZATIONNAME___. All rights reserved.
+//
+//  Template generated by Juanpe Catalán @JuanpeCMiOS
+//
+
+import Foundation
+
+// MARK: Wireframe -
+protocol MovieDetailWireframeProtocol: AnyObject {
+
+}
+// MARK: Presenter -
+protocol MovieDetailPresenterProtocol: AnyObject {
+
+}
+
+// MARK: Interactor -
+protocol MovieDetailInteractorProtocol: AnyObject {
+
+  var presenter: MovieDetailPresenterProtocol? { get set }
+}
+
+// MARK: View -
+protocol MovieDetailViewProtocol: AnyObject {
```

Comment:
Aplica el mismo nombrado de protocolos aquí:

```suggestion
protocol MovieDetailWireframe: AnyObject {

}
// MARK: Presenter -
protocol MovieDetailPresenter: AnyObject {

}

// MARK: Interactor -
protocol MovieDetailInteractor: AnyObject {

  var presenter: MovieDetailPresenter? { get set }
}

// MARK: View -
protocol MovieDetailView: AnyObject {
```